packages/nimble-components/package.json

```diff
@@ -37,7 +38,7 @@
     "test-webkit:debugger": "karma start karma.conf.js --browsers=WebkitDebugging --skip-tags SkipWebkit",
     "test-webkit:watch": "karma start karma.conf.headless.js --browsers=WebkitHeadless --skip-tags SkipWebkit --watch-extensions js",
     "test-webkit": "karma start karma.conf.headless.js --browsers=WebkitHeadless --single-run --skip-tags SkipWebkit",
-    "test-concurrent": "concurrently --pad-prefix --timings --group \"npm:test-chrome\" \"npm:test-webkit\""
+    "test": "concurrently --pad-prefix --timings --group \"npm:test-chrome\" \"npm:test-webkit\""
```

Comment:
Longer term are `concurrently` and `turbo` complementary or will we replace instances of `concurrently` with `turbo`? Would this be equivalent to `turbo test-chrome test-webkit` or something like that?